modules/network/src/main/java/org/apache/ignite/internal/network/configuration/MulticastConfigurationSchema.java

```diff
@@ -42,8 +42,8 @@ public class MulticastConfigurationSchema {
     @Range(min = 0)
     public final int resultWaitTime = 500;
 
-    /** Time to live for multicast packets. */
+    /** Time to live for multicast packets. Value -1 corresponds to system default value. */
```

Comment:
I suggest replacing the value with a reference to a constant, in case its value changes?